kore/test/Test/Kore/Repl/Parser.hs

```diff
@@ -138,6 +138,12 @@ configTests =
                               \expecting end of input, integer, or white space\n"
     ]
 
+leafsTests :: [ParserTest ReplCommand]
```

Comment:
You forgot to add this to the big list on top, so it's not executing yet. A good rule of thumb is to try to make the test so that it fails, for example:

```
"this should fail" `parsesTo` ShowLeafs
```

then run the tests and make sure it fails. You can then change the test to something that works and re-run to make sure the tests pass.